File: Arcade/Intro/C++/11_isLucky.cpp
bool isLucky(int n) {
 string temp = to_string(n);
 int sum1=0;
 int sum2=0;
  for(int i=0;i<temp.length()/2;i++){
    sum1+=temp[i]-'0';
    sum2+=temp[i+(temp.length()/2)]-'0';
  }
  return sum1==sum2;
}

File: Arcade/Intro/JavaScript/11_isLucky.js
function isLucky(n) {
    var arr = n.toString().split('');
    var top = 0, bot = 0, len = arr.length;
    for (var i = 0; i < len/2; i++) {
        bot += parseInt(arr[i], 10);
        top += parseInt(arr[len-i-1], 10);
    } return (bot === top);
}